src/main/java/hotelserviceapp/Support/ExceptionsClass.java

```
@@ -0,0 +1,18 @@
+package hotelserviceapp.Support;
+
+import java.security.InvalidParameterException;
+import java.time.LocalDate;
+
+public class ExceptionsClass  extends RuntimeException{
+	public RuntimeException nullValueException(){
+		return new NullPointerException(" Date variable has null value !");
```

Let's throw a custom exception